Domaci_ukol_Automatizovane_testovani_Cvetelina/src/main/java/com/cvetelina/book/tools/Chapter1Page.java

```diff
@@ -13,199 +13,148 @@
-                }
-            }
-            Assert.assertTrue(match);
+        Select dropDownSelect = new Select(driver.findElement(DROPDOWN_SELECTOR));
+        List<WebElement> options = dropDownSelect.getOptions();
+        for (int i = 0; i < options.size(); i++) {
+            Assert.assertEquals(expectedOptions[i], options.get(i).getText());
         }
-    }
 
-    @Test
-    public void checkDropDownOptionsAreSelectedTest() {
-        Select dropDown = new Select(driver.findElement(By.id("selecttype")));
-        dropDown.selectByVisibleText("Selenium IDE");
-        System.out.println("First option is available and selected value is Selenium IDE.");
-        Assert.assertEquals(dropDown.getFirstSelectedOption().getText(), "Selenium IDE");
-
-        dropDown.selectByIndex(1);
-        System.out.println("Second option is available and selected value is Selenium Core.");
-        Assert.assertEquals(dropDown.getFirstSelectedOption().getText(), "Selenium Core");
-
-        dropDown.selectByIndex(2);
-        System.out.println("Third option is available and selected value is Selenium RC..");
-        Assert.assertEquals(dropDown.getFirstSelectedOption().getText(), "Selenium RC");
-
-        dropDown.selectByIndex(3);
-        System.out.println("Third option is available and selected value is Selenium Grid..");
-        Assert.assertEquals(dropDown.getFirstSelectedOption().getText(), "Selenium Grid");
+        dropDownSelect.selectByIndex(0);
+        Assert.assertEquals(dropDownSelect.getFirstSelectedOption().getText(), "Selenium IDE");
+
+        dropDownSelect.selectByIndex(1);
+        Assert.assertEquals(dropDownSelect.getFirstSelectedOption().getText(), "Selenium Core");
+
+        dropDownSelect.selectByIndex(2);
+        Assert.assertEquals(dropDownSelect.getFirstSelectedOption().getText(), "Selenium RC");
+
+        dropDownSelect.selectByIndex(3);
+        Assert.assertEquals(dropDownSelect.getFirstSelectedOption().getText(), "Selenium Grid");
     }
 
     @Test
-    public void selectCheckBoxtest() {
+    public void selectCheckBoxTest() {
         WebElement checkbox = driver.findElement(By.cssSelector("input[name='selected(1234)']"));
-        for (int i = 0; i < 2; i++) {
-            checkbox.click();
-            if (checkbox.isSelected()) {
-                System.out.println("Checkbox is toggled on.");
-                Assert.assertTrue(checkbox.isSelected());
-            } else {
-                System.out.println("Checkbox is toggled off.");
-                Assert.assertTrue(!checkbox.isSelected());
-            }
-        }
+        Assert.assertTrue(checkbox.isDisplayed());
+        checkbox.click();
+        Assert.assertTrue(checkbox.isSelected());
+        checkbox.click();
+        Assert.assertTrue(!checkbox.isSelected());
     }
 
-    @Test
+    @Test(priority = 1)
     public void homePageLinkTest(){
         String expectedUrl = "http://book.theautomatedtester.co.uk/";
```

Comment:
This is at least 2nd occurrence of this String in this file. Introduce constant. Same at row 95. 

Use constant for general part of URL. Try to place it in some Class where it will be available for the whole project. 

Always try to suppose that elements (things) are going to be reused in project, so if it makes sense, then it is better to define them at 1 place and just reuse.